### PR TITLE
Bug fix for multiple token support in Pushalot plugin

### PR DESCRIPTION
**Pull request by liiight**
Multiple tokens were not configured properly in schema

### CODE REVIEW

File: flexget/plugins/output/pushalot.py [hunk 1]
```diff
@@ -31,12 +31,15 @@ class OutputPushalot(object):
     Configuration parameters are also supported from entries (eg. through set).
     """
     default_body = "{% if series_name is defined %}{{tvdb_series_name|d(series_name)}} " \
-                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} "\
+                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} " \
```
Comment (gazpachoking) [inline flexget/plugins/output/pushalot.py]: Can we just ditch the `\` here? Default python string concatenation or whatever it's called should take care of this, no?

File: flexget/plugins/output/pushalot.py [hunk 2]
```diff
@@ -31,12 +31,15 @@ class OutputPushalot(object):
...
                    "{{imdb_year}}{% else %}{{title}}{% endif %}"
     schema = {
         'type': 'object',
         'properties': {
-            'token': one_or_more({'type': 'string'}),
+            'token': {'oneof': [
+                {'type': 'string'},
+                {'type': 'array', 'items': 'string'}
+            ]},
```
Comment (gazpachoking) [inline flexget/plugins/output/pushalot.py]: Doesn't `one_or_more` produce the same schema?

Comment (liiight) [inline flexget/plugins/output/pushalot.py]: I wasn't actually aware of that, so yeah, keeping it makes perfect sense. I upgraded the schema in the lastest commit to have the ability to use just a string without an explicit token property. Also tested it all using 1 & 2 tokens and everything is peachy.

**Comment by gazpachoking** [comment]
Thanks!

**Pull request merged**